src/tests/ftest/erasurecode/ior_smoke.py

```diff
@@ -25,10 +25,11 @@ def test_ec(self):
             Create the medium size of pool and run IOR with supported EC object
             type class for sanity purpose.
 
-            :avocado: tags=all,pr,daily_regression
-            :avocado: tags=hw,large
-            :avocado: tags=ec,ec_smoke,ior
-            :avocado: tags=ec_ior
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,large
+        :avocado: tags=ec,ec_smoke,ior
+        :avocado: tags=ec_ior
+        :avocado: tags=test_ec
```

Comment:
FYI for this one and others, we usually put these on the same line
```suggestion
        :avocado: tags=ec_ior,test_ec
```

Reply:
Right.  But this work of adding these tags to the many dozens of tests was done before any of that was done or the standard established.

Reply:
I'm also not sure I am understanding why the last two go on the same line but are the last two lines are not combined with the line above them also.  I.e. why not:

```suggestion
        :avocado: tags=all,pr,daily_regression
        :avocado: tags=hw,large
        :avocado: tags=ec,ec_smoke,ior,ec_ior,test_ec
```

So ultimately, I am not sure which ones of the ones I have added should be added to an existing line and which ones should not.  Please advise.

Reply:
> work of adding these tags to the many dozens of tests was done before any of that was done or the standard established

No worries. Not something to block on - just if you need to repush and have bandwidth.


> not sure I am understanding why the last two go on the same line
This shows an example:
https://daosio.atlassian.net/wiki/spaces/DAOS/pages/10984259629/Test+Tags#What-Are-Test-Tags%3F
But...
1. "Feature" tags aren't rigorously defined
2. Lines 3 and 4 are sometimes fuzzy. I.e. it depends on whether you interpret `ec_ior` to be a Feature or a tag unique to this file
3. Many tests are not in compliance

But again, not something to block on.

Reply:
Additionally, should I assume you do not join the tags onto a single line if the line would then exceed the maximum line length?

Reply:
Right. I think it makes sense to break too-long lines